src/dblib/unittests/timeout.c

```diff
@@ -187,6 +179,9 @@ test(int per_process)
 			fprintf(stderr, "Failed: dbsetopt(..., DBSETTIME, \"%d\")\n", timeout_seconds);
 			exit(1);
 		}
+
+		/* Verify setting the global timeout won't override the per-process timeout value */
+		dbsettime(120);
```

Comment:
Is this fails people will wait for quite long, I would use a 35 (query take 30 seconds).